universal-login-relayer/test/testhelpers/startRelayer.ts

```diff
@@ -1,20 +1,20 @@
+import {Wallet} from 'ethers';
 import {deployContract} from 'ethereum-waffle';
-import ENSBuilder from 'ens-builder';
 import {ETHER_NATIVE_TOKEN, deepMerge} from '@unilogin/commons';
 import {deployFactory, beta2} from '@unilogin/contracts';
 import {mockContracts} from '@unilogin/contracts/testutils';
 import {getContractWhiteList} from '../../src/http/relayers/RelayerUnderTest';
-import {getConfig} from '../../src/index';
+import {getConfig, RelayerClass} from '../../src/index';
 
 const defaultDomain = 'mylogin.eth';
 
-async function depolyEns(wallet) {
-  const ensBuilder = new ENSBuilder(wallet);
+async function depolyEns(wallet: Wallet) {
```

Comment:
Do we actually use it somewhere? It doesn't seem to be used, because it shouldn't work  😅